docs/user_guide.rst

```diff
@@ -54,7 +65,8 @@ You can specify the base URL on the command line:
 Using a Configuration File
 --------------------------
 
-You can specify the base URL using a configuration file:
+You can specify the base URL using a
+:ref:`configuration file <configuration-files>`:
```

Comment:
When I view this file on GitHub this doesn't render properly. I assume that's expected and this is only meant to work for rendering readthedocs. Is that correct?